src/components/Sidebar/Sidebar.js

```diff
@@ -203,6 +203,7 @@ const mapStateToProps = state => ({
 
 const mapDispatchToProps = {
   createNewProjectStart: actions.createNewProjectStart,
+  selectProject: actions.selectProject,
```

Comment:
Just wondering when this was removed. Anyway good to have it back - so selecting can work.

Reply:
Was accidentally removed in a change to add no-shadow and no-unused-vars lint rules. Was my bad for not catching it in review